util/dataset.go

```diff
@@ -24,6 +24,13 @@ const (
 	DirIngestPipeline = "ingest-pipeline"
 )
 
+var validTypes = map[string]string{
+	"logs":    "Logs",
```

Comment:
For string-to-string enums there is a different approach recommended: stringer (see: https://godoc.org/golang.org/x/tools/cmd/stringer). 

We can leave it as, it's not a full blown enum